internal/manager/config/config.go

```diff
@@ -635,6 +639,10 @@ func (i *Instance) GetVideoFileNamingAlgorithm() models.HashAlgorithm {
 	return models.HashAlgorithm(ret)
 }
 
+func (i *Instance) GetGalleryCoverRegex() string {
+	return i.getString(GalleryCoverRegex)
+}
```

Comment:
Maybe add an extra check in to make sure that the regex compiles ok and if not log it and fallback to the default one?

Reply:
Yeah that adds a bit of an overhead (compile once here and then once more via the database regex driver), i dont think that is too much though considering the processing is done once for every gallery.
We could move the check later on in the save part of the config ( if the option is added in the UI ) so it will not affect performance or skip that for now and just make sure users check their regexes ( eg mentioning <https://regex101.com/?flavor=golang> , <https://github.com/google/re2/wiki/Syntax> ) if it affects performance (havent actually tested it yet)